src/test/run-make-fulldeps/foreign-double-unwind/Makefile

```diff
@@ -0,0 +1,10 @@
+-include ../tools.mk
+
+all: foo
+	$(call RUN,foo) | $(CGREP) -v unreachable
```

Comment:
Shouldn't this be `! grep unreachable` ? -v inverts the match, so if the program writes _any_ output grep reports success.

We want instead to search for "unreachable" and fail the build if it is found.

Reply:
`$(CGREP)` is not grep.